pandas/core/indexes/datetimelike.py

```diff
@@ -27,10 +28,7 @@
     Tick,
     parsing,
 )
-from pandas._typing import (
-    Callable,
-    final,
-)
+from pandas._typing import Callable
```

Comment:
this looks iffy. we look at this independently and see if this occurs elsewhere.

Reply:
did this get resolved?  if so, LGTM

Reply:
opened #42498